api/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7
```

Comment:
Any specific reason not to use Python 3.8?

Reply:
I'd say at this point in time Python 3.8 is still quite new with limited support by some libraries, while it offers few new language features that encourage upgrading.

Reply:
Ah, so should I use `python:3.7-slim` or `python:3.8-slim`?